iree/compiler/Dialect/HAL/Target/LLVM/LLVMTargetOptions.h

```diff
@@ -38,6 +43,7 @@ struct LLVMTargetOptions {
   // information is valid) it may significantly change the output program
   // and benchmarking
   bool debugSymbols = true;
+  int sanitizerKind = 0;
```

Comment:
I was thinking that it should be converted to other type.. Thanks for letting me know.
And, I changed some design for enum. Moved enum Sanitizer to inside of LLVMTargetOptions and used enum instead of enum class since I thought this sanitizer option would be used only with LLVM Target option.